src/renderer/modules/webpack/index.ts

```diff
@@ -380,6 +380,25 @@ export namespace filters {
       return typeof match === "string" ? source.includes(match) : match.test(source);
     };
   };
+
+  /**
+   * Get a module that has the given value on one of its exports
+   * @param match The string to check the value against
+   *
+   * @remarks
+   * Great for getting a module for a specific classname
+   */
+  export const byValue = (match: string | RegExp) => {
+    return (m: RawModule) => {
+      if (m.exports) {
+        return typeof match === "string"
+          ? Object.values(m.exports).includes(match)
+          : Object.values(m.exports).some((val) => typeof val === "string" && match.test(val));
+      } else {
+        return false;
+      }
```

Comment:
Better to handle the edge cases first
```suggestion
      if (!m.exports) {
        return false
      }

      return typeof match === "string"
        ? Object.values(m.exports).includes(match)
        : Object.values(m.exports).some((val) => typeof val === "string" && match.test(val));
```